elf/elf.go

```diff
@@ -478,9 +478,13 @@ func (b *Module) Load() error {
 		}
 	}
 
-	for name, _ := range b.maps {
+	for name, m := range b.maps {
```

Comment:
could we move this block in a separate function? So the name of the function would make it clear this is about initializing the perf maps, and not just generic elf parsing